.github/workflows/ci.yml

```diff
@@ -19,6 +19,7 @@ env:
   #
 
   # Sourced from https://vulkan.lunarg.com/sdk/home#linux
+  # These Vulkan version definition is duplicated in the install-vulkan action.
   VULKAN_SDK_VERSION: "1.4.321"
   VULKAN_FULL_SDK_VERSION: "1.4.321.0"
```

Comment:
Can we remove this duplication now? If the job needs the version, the composite action can export the env car